### DISCUSSION TITLE
Alternative dosing routes

### DISCUSSION THREAD

**Discussion by ibalshammari** [Open-Systems-Pharmacology/Forum#1014] [Questions & Problems] [answered]
Hello, 
I have encountered an issue to add different administration protocols; for example, I have tried to add inhalation, dermal, or I.P. administration protocols, but I could not. I am wondering if anyone can help me on how to add alternative dosing routes.
Thanks!

**Comment by msevestre** [comment]
This can only be done in MoBi. Please look for instance at the dermal model in the OSP organization to see how you can define a dermal model administration

**Comment by msevestre** [comment, marked as answer]
Here is a link to the skin permeation model
https://github.com/Open-Systems-Pharmacology/Skin-permeation-model

**Reply by ibalshammari** [reply]
Awesome! 
So, can I use MoBi to add the other alternative dosing routes such as inhalation and I.P.?
Thank you so much, Michael.

**Reply by msevestre** [reply]
Yes. But you would need to find the corresponding model extensions for this. Inhalation is far from trivial for instance so I would suggest to look at what's available in the osp organization / publications and build upon this

**Reply by tobiasK2001** [reply]
Sorry for intruding but just to add on to Michael you will find an inhalation model here: https://github.com/Open-Systems-Pharmacology/Inhalation-model.
And indeed it is far from trivial ;-)

Best, Tobias

**Reply by ibalshammari** [reply]
I really appreciate your help. Thank you so much, Michael and Tobias!
